querysync/java/com/google/idea/blaze/qsync/project/ProjectDefinition.java

```diff
@@ -172,7 +172,9 @@ public boolean isExcluded(Path workspacePath) {
    */
   public Optional<Path> getIncludingContentRoot(Path workspacePath) {
     Optional<Path> contentRoot =
-        projectIncludes().stream().filter(workspacePath::startsWith).findAny();
+            projectIncludes().stream()
+                    .filter(projectInclude -> projectInclude.toString().isEmpty() || workspacePath.startsWith(projectInclude))
```

Comment:
@tpasternak what if the file is really outside the workspace, this means that if `directories: .` is used no file will get the "Add file to project" action?